### PR TITLE
diffsims: new recipe

### PR DESCRIPTION
**Pull request by dnjohnstone**

### CODE REVIEW

File: recipes/diffsims/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+  license: GPL-3.0
+  license_family: GPL
+  license_file: LICENSE
+  summary: An open-source Python library for diffraction simulations
+  doc_url: www.pyxem.org
+
+extra:
+  recipe-maintainers:
+    - dnjohnstone
+    - pc494
```
Comment (marcelotrevisani) [inline recipes/diffsims/meta.yaml]: @pc494 could you please confirm that you also want to be a maintainer of this package?

**Comment by marcelotrevisani** [comment]
There is a import error in this project
```
import: 'diffsims'
Traceback (most recent call last):
  File "C:\bld\diffsims_1563267232535\test_tmp\run_test.py", line 2, in <module>
    import diffsims
  File "C:\bld\diffsims_1563267232535\_test_env\lib\site-packages\diffsims\__init__.py", line 25, in <module>
    from .generators.diffraction_generator import DiffractionGenerator
  File "C:\bld\diffsims_1563267232535\_test_env\lib\site-packages\diffsims\generators\diffraction_generator.py", line 26, in <module>
    from diffsims.sims.diffraction_simulation import DiffractionSimulation
ModuleNotFoundError: No module named 'diffsims.sims'

```

**Comment by dnjohnstone** [comment]
> There is a import error in this project
> 
> ```
> import: 'diffsims'
> Traceback (most recent call last):
>   File "C:\bld\diffsims_1563267232535\test_tmp\run_test.py", line 2, in <module>
>     import diffsims
>   File "C:\bld\diffsims_1563267232535\_test_env\lib\site-packages\diffsims\__init__.py", line 25, in <module>
>     from .generators.diffraction_generator import DiffractionGenerator
>   File "C:\bld\diffsims_1563267232535\_test_env\lib\site-packages\diffsims\generators\diffraction_generator.py", line 26, in <module>
>     from diffsims.sims.diffraction_simulation import DiffractionSimulation
> ModuleNotFoundError: No module named 'diffsims.sims'
> ```

Gah - that's irritating, I don't know how on earth that's slipped through the continuous integration on the package, but such is life, will fix it.

File: recipes/diffsims/meta.yaml (hunk 1 above)
Comment (pc494) [reply recipes/diffsims/meta.yaml]: Yep, I am happy to be a maintainer.

**Comment by dnjohnstone** [comment]
@marcelotrevisani thanks for the comments on this - I think this is now good to go?

**Comment by dnjohnstone** [comment]
@conda-forge/staged-recipes I think this is now good to go - please could someone have a look?

**Pull request merged**